### ISSUE TITLE
Getting whole SMPL-X Parameters from Demo.py

### ISSUE THREAD

**Issue by felixshing** [muelea/shapy#28]
Hello, I would like to get the SMPL-X paratmers from demo.py. My target parameters are 
```
'global_orient' (3,), 'transl' (3,), 'body_pose' (63,), 'left_hand_pose' (45,), 'right_hand_pose' (45,), 
 'jaw_pose' (3,), 'leye_pose' (3,), 'reye_pose' (3,), 'expression' (10,), 'betas' (10,)
```

However, after I run demo.py and enable --save-params to get the .npz file, I found the parameters stored in .npz miss some SMPL-X parameters I want, as well as have different shape to what I expect.

This is the data stored in the .npz file after running demo.py 

![image](https://github.com/muelea/shapy/assets/61814345/5c2c2778-5329-4ede-8ee5-bd8c3e445254)

Basically, it contains only betas, transl, global_rot, and body_pose that may be part of what I want. The other parameters are missing. Moreover, I found the the shape of global_rot is (1,3,3) and raw_global_rot is (6,); the shape of body_pose is (21,3,3) and raw_body_pose is (126,). None of these has the same shape with what I expect. What is the meaning of these parameters and how could I get the whole SMPL-X parameters?

**Comment by muelea** [comment]
Hi, 

we use them SMPL-X body model, but do not train hand pose and facial expression. This is why the output contains only global orientation, body pose, body shape, and translation. The 'missing' parameters (expression, hand, eye pose etc.) are set to mean pose (all zeros), so they're not part of the output. 

The different formats for body_pose and global_rot are just conversions between rotation representations. I assume you're expecting axis angle rotation representations for global orientation and body pose. You can convert global_rot which is a rotation matrix (1,3,3) to smpl-x format which is in axis angle (1,3). raw_global_rot is a 6D rotation representation which we use for training (see this [paper](https://openaccess.thecvf.com/content_CVPR_2019/papers/Zhou_On_the_Continuity_of_Rotation_Representations_in_Neural_Networks_CVPR_2019_paper.pdf)).

**Comment by felixshing** [comment]
> Thank you for your reply. If I want to get the full parameters, what tools I can use, for example, smplify-x? I am actually using a rgb-d cameras thus I can still provide depth information.

I have successfully used smplify-x to get the SMPL-X parameters. However, the speed is extremely low... Is there any faster solution to get this? BTW, I found the translation parameter got back from smplify-x is "camera_translation". Is it the same as the  "transl" used in shapy and the example I proposed in my first question?

**Comment by felixshing** [comment]
Another question is related to the scale of shapy and smplify-x. In the follwing meshlab environment, I plot the mesh obtained from shapy (the leftmost), the mesh obtained from smplify-x (the rightmost), and the point cloud obtained from rgb-d image (the middle). As we can see, the mesh obtained from smplify-x lies in the origin. However, the mesh from shapy deviates a lot, in terms of z-axis. Moreover, both shapy and smplify-x deviate y-axis a little bit. How to solve this problem? Is it because shapy and smplify-x lacks of depth information?

![image](https://github.com/muelea/shapy/assets/61814345/735be931-1722-4059-b9c4-6a2af6e45dca)

**Comment by phongnhhn92** [comment]
@felixshing hey have you found the answer why we have a mismatch between camera params of shapy and smplify-x ? I am also having the same issue.
